source/elements/oneDAL/source/common_interface/common_types.rst

```diff
@@ -0,0 +1,29 @@
+============
+Common types
+============
+
+This section describes common types of |dal_short_name|.
```

Comment:
"common types" or "common types of oneDAL" -- this is not a clear term. Common types of what? Because if you say "common types of oneDAL", it means there are different types of oneDAL itself. Common types of objects in oneDAL? Common data types?